src/api.js

```diff
@@ -708,12 +709,17 @@ API.prototype.router = function(options) {
                     "supported; use remote signature validation");
   }
 
-  // Create statistics reporter
+  // Create statistics reporters
   var reporter = null;
   if (options.drain) {
     assert(options.component, "The component must be named in statistics!");
     reporter = series.ResponseTimes.reporter(options.drain);
   }
+  var monitor = null;
+  if (options.monitor) {
+    assert(options.component, "The component must be named for monitoring!");
+    monitor = options.monitor.prefix(options.component + '.api');
```

Comment:
or, how about we do this inside the loader filer for each component... that way it's more transparent?

I guess we would be repeating it, but then we don't need to pass `options.component` anymore do we?